packages/twenty-front/src/modules/object-record/record-filter/utils/isRecordMatchingFilter.ts

```diff
@@ -160,15 +160,15 @@ export const isRecordMatchingFilter = ({
     }
 
     switch (objectMetadataField.type) {
-      case FieldMetadataType.Select:
       case FieldMetadataType.Rating:
-      case FieldMetadataType.MultiSelect:
       case FieldMetadataType.Text: {
         return isMatchingStringFilter({
           stringFilter: filterValue as StringFilter,
           value: record[filterKey],
         });
       }
+      case FieldMetadataType.Select:
+      case FieldMetadataType.MultiSelect:
       case FieldMetadataType.Array: {
         return isMatchingArrayFilter({
           arrayFilter: filterValue as ArrayFilter,
```

Comment:
this looks wrong to me (see comment above) why would Select checking isMatchingArrayFilter. MultiSelect is already borderline to me (as constainsILike does not make sense), Select seems wrong